css/styles.css

```diff
@@ -192,12 +254,58 @@ body {
   text-transform: uppercase;
 }
 
+/*.input--submit:hover {
+  background-color: thistle;
+}*/
+
+.input--submit-answers { 
+  background-color: red;
+  color: white;
+}
+
+.input--submit-answers:hover {
+  background-color: green;
+}
+
 .input--submit-names {
+  grid-column: 2/3;
   height: 36px;
-  width: 440px;
+  width: 200px;
+  opacity: .8;
+  justify-self: center;
   border-radius: 18px;
-  background-color: #2e1d75;
-  color: #ffffff;
+  color: thistle;
+  margin-top: 10px;
+}
+
+.button--exit {
+  background-color: red;
+}
+
+.button--exit:hover {
+  background-color: green;
+}
+
+.input--submit-names:hover {
+  opacity: 1;
+  color: lightskyblue;
+}
+
+.em {
+  display: block;
+  margin: auto;
+  position: absolute;
+  font-size: 20px;
+}
+
+.em-top-right {
+  top: 5px;
+  right: 5px;
+}
+
+.em-bottom {
+  bottom: 5px;
+  left: 5px;
```

Comment:
These are the suit emojis. We definitely need to way to have them adjust for the longer category names. Let me know a better way of doing this.